docs/integrations/wallets/ledger/ledger-native.md

```diff
@@ -2,7 +2,108 @@
 sidebar_position: 10
 title: Ledger for Astar Native Accounts
 ---
+import ledger1 from "./img/native/ledger1.png"
+import ledger2 from "./img/native/ledger2.png"
+import ledger3 from "./img/native/ledger3.png"
+import ledger4 from "./img/native/ledger4.png"
+import ledger5 from "./img/native/ledger5.png"
+import ledger6 from "./img/native/ledger6.png"
```

Comment:
Let's use this image instead :D

https://user-images.githubusercontent.com/92044428/237559813-3088b489-f953-4676-bc06-2bd437b305ee.png

Reply:
(maybe you can hide the wallet address)